storage/container_storage_interface/persistent-storage-csi-azure-file.adoc

```diff
@@ -21,12 +21,12 @@ Azure File CSI Driver Operator does not support:
 
 * Virtual hard disks (VHD)
 
-* Network File System (NFS): {product-title} does not deploy a NFS-backed storage class.
-
-* Running on nodes with FIPS mode enabled.
+* Running on nodes with Federal Information Processing Standard (FIPS) mode enabled for Server Message Block (SMB) file share. However, Network File System (NFS) does support FIPS mode.
```

Comment:
Do we need to say anything about SMB (thats a different FS type) ?

Reply:
I've added an emphasis on the "not" to make this clearer.